scripts/actions/mobskills/Orthopterror.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+-- Name: Orthopterror
+-- Description: Delivers a threefold attack, Conal AoE, inflicts Terror and Knockback.
+-- Type: Physical
+-- Utsusemi/Blink absorb: 3 shadows
+-----------------------------------
+local mobskillObject = {}
```

Comment:
For all of the mobskill scripts, please add `---@type TMobSkill` above this definition